sdk/nodejs/tests/automation/remoteWorkspace.spec.ts

```diff
@@ -14,7 +14,199 @@
 
 import assert from "assert";
 
-import { isFullyQualifiedStackName } from "../../automation";
+import {
+    fullyQualifiedStackName,
+    isFullyQualifiedStackName,
+    LocalWorkspace,
+    RemoteGitAuthArgs,
+    RemoteGitProgramArgs,
+    RemoteStack,
+    RemoteWorkspace,
+    RemoteWorkspaceOptions,
+} from "../../automation";
+import { asyncTest } from "../util";
+import { getTestOrg, getTestSuffix } from "./util";
+
+const testRepo = "https://github.com/pulumi/test-repo.git";
+
+describe("RemoteWorkspace", () => {
+    describe("selectStack", () => {
+        describe("throws appropriate errors", () => testErrors(RemoteWorkspace.selectStack));
+    });
+
+    describe("createStack", () => {
+        describe("throws appropriate errors", () => testErrors(RemoteWorkspace.createStack));
+
+        it(`runs through the stack lifecycle`, async function () {
```

Comment:
Unfortunately, this test takes ~3 minutes. What can we do to avoid running it in PRs?

Reply:
We'll disable the test by default (requires `PULUMI_TEST_DEPLOYMENTS_API` to be set). And consider running the test in a cron until we add support for mocking the deployments api.